pvnet/models/base_model.py

```diff
@@ -94,7 +94,7 @@ def minimize_data_config(input_path, output_path, model):
         if not model.include_nwp:
             del config["input_data"]["nwp"]
         else:
-            for nwp_source in config["input_data"]["nwp"].keys():
+            for nwp_source in list(config["input_data"]["nwp"].keys()):
```

Comment:
Out of interest why has list been added? Is there a difference between looping through the items in the list vs the view object or something else?

Reply:
Yeh good point. I had that change locally for some reason I've forgotten. I'm pretty sure you're right, so I'll change back